po/ar/minetest.po

```diff
@@ -1277,8 +1277,8 @@ msgid "Camera update enabled"
 msgstr "تحديث الكاميرا مفعل"
 
 #: src/client/game.cpp
-msgid "Can't show block bounds (need 'basic_debug' privilege)"
-msgstr "تعذر إظهار حدود الكتل ( تحتاج امتياز 'basic_debug')"
+msgid "Can't show block bounds (disabled by mod or game)"
+msgstr ""
```

Comment:
Please don't change .po files in files, as it conflicts with translation updates